.bash_profile

```diff
@@ -1,3 +1,9 @@
+# Prefer GNU coreutils over BSD ones, i.e. ls, cat. This works only if you run 'brew install coreutils', or run brew.sh
+PATH=$(brew --prefix coreutils)/libexec/gnubin:$PATH
+# Allow access to GNU coreutils man pages
+MANPATH=/usr/local/opt/coreutils/libexec/gnuman:${MANPATH}
+export MANPATH
```

Comment:
+1, and please also consider what happens if `brew` is not installed (e.g. on Linux boxes over `ssh`).